_docs/dev-licence.md

```diff
@@ -0,0 +1,24 @@
+## 🧑‍🎓 TinaCMS Dev Licence
+
+We encourage all developers to get their “TinaCMS licence” - that is, to get it up and running and complete the basics before implementing TinaCMS on a real project. However, this is a suggestion for those who wish to deepen their knowledge.  
+Here are the steps to complete your journey:
+
+1. Join [Tina's Discord channel](https://discord.com/invite/zumN63Ybpf) - You can leave a message on the `#introductions`, i.e.:
+
+    > Hi, I am trying out TinaCMS with {{ TECH eg. NextJS }}.  
+    > After I have evaluated TinaCMS, I plan to use it in the {{ INDUSTRY }} industry.  
+    > Previous CMSs I have used include {{ FILL IN }}.
+
+2. Create a Tina instance using the default template on [tina.io](https://tina.io) and [TinaCloud](https://app.tina.io/signin).
+
+3. Record a [Done Video](https://www.ssw.com.au/rules/record-a-quick-and-dirty-done-video/) explaining how Tina works.
+
+    Here is a sample Done Video: https://www.youtube.com/watch?v=OhVYTOKCsWI  
```

Comment:
Is this the video we want to use? Should we change this to a Tina related video? (Maybe someone doing this exact process?)

Reply:
dropped it, didn't make sense in the context of the video we want
added a list of things for us to look out for instead